recipes/firefox/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+APP=${PREFIX}/bin/firefox
```

Comment:
The APP is `${PREFIX}/bin/Firefox/firefox`

Reply:
Oh never mind. I see this is the "app" as in the bash script that launches the app.